code/src/components/GameScreen.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
It's great to see that the code is well-structured and easy to read. The component's purpose is clear, and the code is well-indented and formatted. It's essential to maintain this consistency throughout the codebase.